_announcements/2021-02-02-announcement.md

```diff
@@ -0,0 +1,24 @@
+---
+title: February 2021 NREL HPC Monthly Update
+data: 2021-02-02
+layout: default
+brief: Eagle outage, Jupyterhub documentation, ESIF-HPC-3 update
+---
+
+# Eagle Outage
+
+Eagle will be taking an outage on March 2nd. We will be making time critical hardware repairs 
+to the Lustre file system and doing some security patching. Small tweaks are being made to slurm to 
+reduce the amount of advertised memory and reduce the number of E-cells jobs run on.  Both should make 
+nodes and jobs more reliable and increase performance for larger jobs.
+
+
+#Jupyterhub Documentation
```

Comment:
Small issue, but need a space after # to render correctly.

Reply:
Apologies for the typo. Updated.